app/src/main/AndroidManifest.xml

```diff
@@ -87,7 +87,7 @@
       android:name=".home.overdue.phonemask.PhoneMaskBottomSheet"
       android:theme="@style/Clinic.V2.Theme.BottomSheetActivity"
       android:windowSoftInputMode="adjustResize" />
-    <activity android:name=".scheduleappointment.PatientFacilityChangeActivity" />
+    <activity android:name=".scheduleappointment.patientFacilityTransfer.PatientFacilityChangeActivity" />
```

Comment:
Accidental break in package naming conventions? (`patientFacilityTransfer` is in camel-case, which is not the usual convention).

Reply:
Oops. Will fix this.